src/test/java/org/springframework/data/couchbase/domain/Address.java

```diff
@@ -26,4 +44,20 @@ public void setCity(String city) {
 		this.city = city;
 	}
 
+	public String getParent_id() {
```

Comment:
why mix underscore and camel case in that file?

Reply:
Fixed.